spec/lib/saml_idp/configurator_spec.rb

```diff
@@ -47,5 +47,34 @@ module SamlIdp
     it 'has a valid session_expiry' do
       expect(subject.session_expiry).to eq(0)
     end
+
+    context "logger initialization" do
+      context 'when Rails has been properly initialized' do
+        it 'sets logger to Rails.logger' do
+          stub_const("Rails", Class.new)
+          allow(Rails).to receive(:respond_to?).with(:logger).and_return(true)
+          allow(Rails).to receive(:logger).and_return(double("Rails.logger"))
```

Comment:
```suggestion
          stub_const("Rails", double("Rails", logger: double("Rails.logger"))
```
maybe somehow like that? 🤔